.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 <!--- Provide a brief summary of the issue in the title above -->
+<!--- We'd kindly ask you to open any issues related to the Gradle Android plugin or Android studio on the Google forum at `https://source.android.com/source/report-bugs` -->
```

Comment:
Can you elaborate on your second suggestion? I think users would still have select between `Android Public Tracker > App Development > Android Studio > Gradle IDE` and `Android Public Tracker > App Development > Gradle`. If those options change then our suggestion would get out of date. I am voting for not going into details. The page we reference explains all details.